PostToExternalServices.py

```diff
@@ -0,0 +1,156 @@
+import json
+import logging
+import os
+
+import boto3
+import requests
+import tweepy
+
+AUTH0_BASE = "https://calligre.auth0.com"
+AUTH0_API = "%s/api/v2/" % AUTH0_BASE
```

Comment:
Does the trailing slash matter? Elsewhere you're doing `'{}/path/to/route'.format(AUTH0_API)`.